cloudify_nsx/network/esg.py

```diff
@@ -94,7 +95,6 @@ def create(**kwargs):
                 "Edge '%s' already exists" % edge_dict["name"]
             )
 
-    if not resource_id:
```

Comment:
Looks as when we have use_existing==True and {'edge': { 'name': 'some_thing'}} - we will try to recreate edge.